layouts/partials/header.html

```diff
@@ -11,12 +11,34 @@
                     </button>
                 </li>
                 <ul id="hamburger-menu" class="hidden md:flex md:flex-nowrap md:space-x-8 md:space-y-0 space-y-4">
-                    {{ range site.RegularPages }}
-                        <li><a class="hover:underline" href="{{ .RelPermalink }}">{{ .LinkTitle }}</a></li>
+                    {{ $currentPage := . }}
+                    {{ range .Site.Menus.main }}
```

Comment:
Isn't this approach a difficult one for a new user?

While this might give us more options, isn't it better to add this details in page front matter than to in the config?

Reply:
I do not know much about Hugo and I am learning. But, I think maybe it is better to go with the standard at first. 

I think New users will eventually have to learn Hugo the way it is recommended by its creators.